apps/web/src/pages/brand/tabs/LayoutEditor.tsx

```diff
@@ -95,77 +125,134 @@ export function LayoutEditor({
-                      data-test-id="is-default-layout"
-                      label="Set as Default"
+                      mb={30}
+                      data-test-id="layout-description"
+                      label="Layout Description"
+                      placeholder="Describe your layout..."
                     />
-                  );
-                }}
-              />
-            </Center>
-          </Grid.Col>
-        </Grid>
+                  )}
+                />
+              </Grid.Col>
+              <Grid.Col md={2} sm={12}>
+                <Center>
+                  <Controller
+                    name="isDefault"
+                    control={control}
+                    render={({ field }) => {
+                      return (
+                        <Checkbox
+                          checked={field.value === true}
+                          disabled={readonly}
+                          onChange={field.onChange}
+                          mt={30}
+                          data-test-id="is-default-layout"
+                          label="Set as Default"
+                        />
+                      );
+                    }}
+                  />
+                </Center>
+              </Grid.Col>
+            </Grid>
 
-        <Controller
-          name="content"
-          data-test-id="layout-content"
-          control={control}
-          render={({ field }) => {
-            return <EmailCustomCodeEditor onChange={field.onChange} value={field.value} />;
+            <Controller
+              name="content"
+              data-test-id="layout-content"
+              control={control}
+              render={({ field }) => {
+                return <EmailCustomCodeEditor onChange={field.onChange} value={field.value} />;
+              }}
+            />
+            <Button submit mb={20} mt={25} data-test-id="submit-layout">
+              {editMode ? 'Update' : 'Create'}
+            </Button>
+          </form>
+        </Grid.Col>
+        <Grid.Col
+          span={3}
+          style={{
+            maxWidth: '350px',
           }}
-        />
-        <Button submit mb={20} mt={25} data-test-id="submit-layout">
-          {editMode ? 'Update' : 'Create'}
-        </Button>
-      </form>
+        >
+          <VariablesManagement
+            index={0}
+            openVariablesModal={() => {
+              setModalOpen(true);
+            }}
+            path="variables"
+            control={control}
+          />
+        </Grid.Col>
+      </Grid>
+      <Modal
+        opened={modalOpen}
+        overlayColor={theme.colorScheme === 'dark' ? colors.BGDark : colors.BGLight}
+        overlayOpacity={0.7}
+        styles={{
+          modal: {
+            backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
+            width: '90%',
+          },
+          body: {
+            paddingTop: '5px',
+            paddingInline: '8px',
+          },
+        }}
```

Comment:
Should we use `useStyles`?

Reply:
I'm not sure I like doing that for a very specific Modal. We already have a Modal in our design system. I would prefer to make that one more flexible to style changes. Will add it to backlog ❤️ Maybe a community member would like to  help with thah.